components/com_users/tmpl/reset/default.php

```diff
@@ -29,7 +29,7 @@
             </h1>
         </div>
     <?php endif; ?>
-    <form id="user-registration" action="<?php echo Route::_('index.php?option=com_users&task=reset.request'); ?>" method="post" class="com-users-reset__form form-validate form-horizontal well">
+    <form id="user-registration" action="<?php echo Route::_('index.php?task=reset.request'); ?>" method="post" class="com-users-reset__form form-validate form-horizontal well">
```

Comment:
```suggestion
    <form action="<?php echo Route::_('index.php?task=reset.request'); ?>" method="post" id="user-registration" class="com-users-reset__form form-validate form-horizontal well">
```

action/method/name/id/class

Reply:
Thank you!